cedar-policy/CHANGELOG.md

```diff
@@ -22,6 +22,9 @@ Cedar Language Version: TBD
   format. The new functions are deprecated and placed behind the `deprecated-schema-compat` feature. (#1600)
 - `Expression::new_duration`, `Expression::new_datetime`, `RestrictedExpression::new_duration`,
    and `RestrictedExpression::new_datetime` (#1614)
+- Added a function to be able to split a policy set parsed from a large file into its component static policies
```

Comment:
Why say "policy set parsed from a large file"? You only need a `PolicySet` and it doesn't need to be large.